authorization/permission/repository/privilege_cache.go

```diff
@@ -139,16 +139,7 @@ func (m *GormPrivilegeCacheRepository) Create(ctx context.Context, privilegeCach
 func (m *GormPrivilegeCacheRepository) Save(ctx context.Context, privilegeCache *PrivilegeCache) error {
 	defer goa.MeasureSince([]string{"goa", "db", "privilege_cache", "save"}, time.Now())
 
-	obj, err := m.Load(ctx, privilegeCache.PrivilegeCacheID)
-	if err != nil {
-		log.Error(ctx, map[string]interface{}{
-			"privilege_cache_id": privilegeCache.PrivilegeCacheID,
-			"err":                err,
-		}, "unable to update privilege cache")
-		return errs.WithStack(err)
-	}
-
-	err = m.db.Model(obj).Updates(privilegeCache).Error
+	err := m.db.Save(privilegeCache).Error
```

Comment:
good point, there's no need to check if the record already exists ;)

Reply:
This change was only made because Gorm doesn't support updates for boolean values.  See [this issue](https://github.com/jinzhu/gorm/issues/202) as one example (there are others).